src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/reports/EnvelopeCountSummaryRepository.java

```diff
@@ -15,18 +15,26 @@
     @Query(
         nativeQuery = true,
         value = "SELECT\n"
-            + "  container,\n"
-            + "  date,\n"
+            + "  first_events.container AS container,\n"
+            + "  date(first_events.createdat) AS date,\n"
             + "  count(*) AS received,\n"
-            + "  SUM(CASE WHEN event IN ('DOC_FAILURE', 'FILE_VALIDATION_FAILURE', 'DOC_SIGNATURE_FAILURE') THEN 1 ELSE 0 END) AS rejected\n"
+            + "  SUM(CASE WHEN rejection_events.id IS NOT NULL THEN 1 ELSE 0 END) AS rejected\n"
             + "FROM (\n"
             + "  SELECT DISTINCT on (container, zipfilename)\n"
-            + "  container, zipfilename, date(createdat), event\n"
+            + "    container, zipfilename, createdat\n"
             + "  FROM process_events\n"
-            + "  ORDER BY container, zipfilename, date(createdat) ASC\n"
+            + "  ORDER BY container, zipfilename, createdat ASC\n"
```

Comment:
Why is date(...) dropped?

Reply:
You're right, this is necessary.

Reply:
Having had more thoughts, we should probably remove createdat from result of this subquery (we should still order by it) and in the main SELECT we should just include :date from the parameter.